packages/marshal/src/encodePassable.js

```diff
@@ -298,7 +315,9 @@ const encodeTagged = (tagged, encodePassable) =>
 
 const decodeTagged = (encoded, decodePassable) => {
   assert(encoded.startsWith(':'));
-  const tagpayload = decodeArray(encoded.substring(1), decodePassable);
+  // Skip the ":" inside `decodeArray` to avoid slow `substring` in XS.
+  // https://github.com/endojs/endo/issues/1984
+  const tagpayload = decodeArray(encoded, decodePassable, 1);
```

Comment:
```suggestion
  const tagPayload = decodeArray(encoded, decodePassable, 1);
```

Or do you have a convention for when you avoid midcaps? Why? I also notice `keysvals` above.

Reply:
No, it was preexisting (and ~~blame~~ praise) identifies introduction in the landmark https://github.com/Agoric/agoric-sdk/pull/4468). But worth fixing here, which I have done for both sites.

Reply:
(FWIW I'm surprised I did that and have no idea why I did. Glad to see it fixed, thanks!)